blueprint-test-utils/src/test_ext.rs

```diff
@@ -372,14 +446,14 @@ pub async fn new_test_ext<
 }
 
 pub struct LocalhostTestExt {
-    client: OnlineClient<SubstrateConfig>,
+    client: OnlineClient<TangleConfig>,
```

Comment:
```suggestion
    client: TangleClient,
```
The `TangleClient` and `TangleConfig` types should be used everywhere possible.